pkg/upstream/http.go

```diff
@@ -92,6 +95,19 @@ func (h *httpUpstreamProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 	}
 }
 
+type unixRoundTripper struct {
+	Transport *http.Transport
+}
+
+func (t *unixRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
```

Comment:
Missing godoc, would be good to explain how this differs from regular http transports

Reply:
done :)